### PR TITLE
[Fix] Quick fix for document model

### PR DESCRIPTION
**Pull request by christorange**
# Description
A quick fix for document models

## Main changes explained:
- Add status, feedback entities to Document model
- Change document fields of Profile to valid format

### CODE REVIEW

File: Server/models/Profile.model.js [hunk 1]
```diff
@@ -166,50 +166,18 @@ const ProfileSchema = new Schema({
...
     type: refType,
     ref: "Document",
-    status: {
-      type: String,
-      enum: constants.documentStatus,
-      default: "PENDING",
-    },
-    feedback: {
-      type: String,
-    },
```
Comment (yuelikestocode) [inline Server/models/Profile.model.js:176]: `status` and `feedback` are still useful right?

How about make it a nested document
```
OPTReceipt: {
    document: {
      type: refType,
      ref: "Document",
    },
    status: {
      type: String,
      enum: constants.documentStatus,
      default: "PENDING",
    },
    feedback: {
      type: String,
    },
  },
```

**Comment by yuelikestocode** [comment]
Would you also do me a favor and change the `Document` model in this PR?

```js
import mongoose from "mongoose";

const Schema = mongoose.Schema;

const DocumentSchema = new Schema({
  URL: {
    type: String,
  },
  S3Bucket: {
    type: String,
    required: true,
    default: "bgp-zichan",
  },
  S3Key: {
    type: String,
    required: true,
  },
});

export default mongoose.model("Document", DocumentSchema);

```

@christorange

File: Server/models/Profile.model.js (hunk 1 above)
Comment (christorange) [reply Server/models/Profile.model.js:176]: Yes I think this will make backend logic easier to write

**Comment by christorange** [comment]
np, done

**Review by yuelikestocode — APPROVED** [review]
Looks good!

**Review by ZichanYang324 — APPROVED** [review]
Well done, merging

**Pull request merged**